pkg/scheduler/framework/plugins/foo/foo.go

```diff
@@ -0,0 +1,35 @@
+package foo
+
+import (
+	"context"
+
+	clusterv1alpha1 "github.com/karmada-io/karmada/pkg/apis/cluster/v1alpha1"
+	policyv1alpha1 "github.com/karmada-io/karmada/pkg/apis/policy/v1alpha1"
+	workv1alpha2 "github.com/karmada-io/karmada/pkg/apis/work/v1alpha2"
```

Comment:
it is better to  move this file foo.go to the folder examples

Reply:
Sorry for the wrong commit. Removed.